apps/cowswap-frontend/src/common/pure/TradeLoadingButton/index.tsx

```diff
@@ -3,31 +3,10 @@ import { useEffect, useState } from 'react'
 import { LONG_LOAD_THRESHOLD } from '@cowprotocol/common-const'
 
 import { Text } from 'rebass'
-import styled from 'styled-components/macro'
 
-import { Dots } from 'legacy/components/swap/styleds'
 import { ThemedText } from 'legacy/theme'
 
-const fadeIn = `
-  @keyframes fadeIn {
-    0% {
-      opacity: 0;
-    }
-    100% {
-      opacity: 1;
-    }
-  }
-`
-
-const CenteredDots = styled(Dots)<{ smaller?: boolean }>`
-  vertical-align: ${({ smaller = false }) => (smaller ? 'normal' : 'super')};
-`
-
-const LongLoadText = styled.span`
-  animation: fadeIn 0.42s ease-in;
-
-  ${fadeIn}
-`
+import { CenteredDots, LongLoadText } from 'modules/trade/pure/TradeConfirmation/styled'
```

Comment:
Common things should not depend on modules.
Would be event better to move the component into `@cowprotocol/ui`

Reply:
@shoom3301 addressed this now